cocos/renderer/backend/opengl/TextureGL.cpp

```diff
@@ -55,9 +64,12 @@ namespace
         }
     }
     
-    GLint toGLAddressMode(SamplerAddressMode addressMode)
+    GLint toGLAddressMode(SamplerAddressMode addressMode, bool bPow2)
     {
-        GLint ret = GL_REPEAT;
+        GLint ret = GL_CLAMP_TO_EDGE;
+        if(!bPow2)
+            return ret;
```

Comment:
It is better to output a log to say that the address mode is changed to `GL_GLAMP_TO_EDGE`.

Reply:
Done!